Makefile

```diff
@@ -85,3 +85,35 @@ functional-test: ## Run functional tests (needs build-functional-test-image)
 .PHONY: bench
 bench: go-generate ## Run benchmark test. See https://pkg.go.dev/cmd/go#hdr-Testing_flags
 	go test ./... -bench . -benchtime 5s -timeout 0 -run=XXX -cpu 1 -benchmem
+
+.PHONY: test-migration-postgres
+test-migration-postgres:
```

Comment:
The Makefile is meant to be long-lived utilities for development over the lifetime of the project. Since these steps are short-lived just for the lifetime of this PR and rollout process for it, then can we please include these as Github Gist snippets in the description of this PR instead of in the Makefile?

Reply:
Yep